### ISSUE TITLE
Don't clear code when switching pages

### ISSUE THREAD

**Issue by tcosprojects** [levibotelho/LearnAngular#15]
On one of the lessons I started writing my solution and then wanted to go back and review one of the previous lessons. When I went back to my code it was all gone. It would be nice if this state wasn't cleared.

**Comment by aggarwalankush** [comment]
Yes, that would be a good feature. Also, it would be great if links open in new tab instead of same window.

**Comment by erikeckhardt** [comment]
Additionally, save the code in a cookie so upon return, all the user's previous entries are still in place.
